### PR TITLE
chore: alpha to master for release 2.1

### PR DESCRIPTION
**Pull request by remisultan**
Please wait for https://github.com/gravitee-io/gravitee-ae-connectors/pull/17 to be merged

### CODE REVIEW

File: docker/integration-tests/one-apim_two-ae/.env.template
Comment (passionne) [inline docker/integration-tests/one-apim_two-ae/.env.template:1]: I don't think it is part of that PR 😊

Comment (remisultan) [reply docker/integration-tests/one-apim_two-ae/.env.template:1]: I disagree 
image

**Pull request merged**